master/internal/api_ntsc_intg_test.go

```diff
@@ -318,6 +318,9 @@ func TestAuthZCanCreateNSC(t *testing.T) {
 	).Times(3)
 	workspaceAuthZ.On("CanGetWorkspace", mock.Anything, mock.Anything, mock.Anything).
 		Return(nil).Times(3)
+	mockRM := MockRM()
+	mockRM.On("SmallerValueIsHigherPriority", mock.Anything).Return(true, nil)
+	api.m.rm = mockRM
```

Comment:
This is a small nit but you can initialize the mockRM earlier with
```
mockRM := MockRM()
api, authZExp, _, curUser, ctx := setupExpAuthTest(t, nil, mockRM)
```

And then do just the
 ```
mockRM.On("SmallerValueIsHigherPriority", mock.Anything).Return(true, nil)
```
here!